tests/e2e/tests/MochaHooks.ts

```diff
@@ -0,0 +1,47 @@
+
+import { IDriver } from '..';
+import { TYPES, CLASSES } from '..';
+import { EditorType, TestConstants } from '../TestConstants';
+import { AskForConfirmationTypeTheia, PreferencesHandlerTheia, TerminalRendererTypeTheia } from '../utils/theia/PreferencesHandlerTheia';
+import { CheApiRequestHandler } from '../utils/requestHandlers/CheApiRequestHandler';
+import { TimeoutConstants } from '../TimeoutConstants';
+import * as monacoPageObjects from 'monaco-page-objects';
+import * as vscodeExtensionTesterLocators from 'vscode-extension-tester-locators';
+import { e2eContainer } from '../inversify.config';
+
+const driver: IDriver = e2eContainer.get(TYPES.Driver);
+const preferencesHandler: PreferencesHandlerTheia = e2eContainer.get(CLASSES.PreferencesHandlerTheia);
+
+exports.mochaHooks = {
+    beforeAll: [
+        async function enableRequestInterceptor() {
+            if (TestConstants.TS_SELENIUM_REQUEST_INTERCEPTOR) {
+                CheApiRequestHandler.enableRequestInteceptor();
+            }
+        },
+        async function enableResponseInterceptor() {
+            if (TestConstants.TS_SELENIUM_RESPONSE_INTERCEPTOR) {
+                CheApiRequestHandler.enableResponseInterceptor();
+            }
+        },
+        async function setTheiaSpecificPreferences() {
+            if (TestConstants.TS_SELENIUM_EDITOR === EditorType.THEIA) {
+                await preferencesHandler.setConfirmExit(AskForConfirmationTypeTheia.never);
+                await preferencesHandler.setTerminalType(TerminalRendererTypeTheia.dom);
+            }
+        },
+        async function initMonacoPageObjects() {
+            if (TestConstants.TS_SELENIUM_EDITOR === EditorType.CHE_CODE) {
+                // init vscode-extension-tester monaco-page-objects
+                monacoPageObjects.initPageObjects(TestConstants.TS_SELENIUM_MONACO_PAGE_OBJECTS_USE_VERSION, TestConstants.TS_SELENIUM_MONACO_PAGE_OBJECTS_BASE_VERSION, vscodeExtensionTesterLocators.getLocatorsPath(), driver.get(), 'google-chrome');
+            }
+        },
+        async function prolongTimeoutConstantsInDebugMode() {
+            if (TestConstants.TS_LOCAL_DEBUG_MODE === true) {
```

Comment:
So that it should be helpful when debug the test remotaly using VNC server, I would propose to rename `TS_LOCAL_DEBUG_MODE` to more general `TS_DEBUG_MODE`.

Reply:
Renamed

Reply:
thanks